From ff821343e61f5cac0fc0d5122ae6214a7378a0e6 Mon Sep 17 00:00:00 2001
From: "Ilja H. Friedel" <ihf@chromium.org>
Date: Tue, 13 Jun 2023 00:04:28 -0700
Subject: [PATCH] Drop Elkhart Lake PCI id 0x4555

Elkhart Lake PCI id 0x4555 is known to cause problems (b/283890347).
(It is best to never drop this patch.)
---
 media_driver/linux/gen11/ddi/media_sysinfo_g11.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/media_driver/linux/gen11/ddi/media_sysinfo_g11.cpp b/media_driver/linux/gen11/ddi/media_sysinfo_g11.cpp
index 7001fcdfb..e4a106eac 100644
--- a/media_driver/linux/gen11/ddi/media_sysinfo_g11.cpp
+++ b/media_driver/linux/gen11/ddi/media_sysinfo_g11.cpp
@@ -435,6 +435,3 @@ static bool ehlDevice4E71 = DeviceInfoFactory<GfxDeviceInfo>::
 
 static bool ehlDevice4E55 = DeviceInfoFactory<GfxDeviceInfo>::
     RegisterDevice(0x4E55, &ehlDevInfo16EU);
-
-static bool ehlDevice4555 = DeviceInfoFactory<GfxDeviceInfo>::
-    RegisterDevice(0x4555, &ehlDevInfo16EU);
-- 
2.41.0.162.gfafddb0af9-goog

